lib/cadet_web/admin_controllers/admin_grading_controller.ex

```diff
@@ -249,6 +235,20 @@ defmodule CadetWeb.GradingController do
     response(401, "Unauthorised")
   end
 
+  swagger_path :grading_summary do
```

Comment:
Added Swagger documentation for the `/grading/summary` endpoint, which was previously undocumented.